src/features/course-main/course-main.test.tsx

```diff
@@ -73,3 +60,22 @@ describe('CourseMain', () => {
     expect(imageElement).toHaveAttribute('src', MOCKED_IMAGE_PATH);
   });
 });
+
+describe('CourseMain', () => {
+  it('renders the section with correct label depending on date', () => {
+    (useCourseByTitle as Mock).mockReturnValueOnce({
+      ...testCourse,
+      course: {
+        ...testCourse.course,
+        startDate: new Date().toLocaleString('en-US', {
+          year: 'numeric',
+          month: 'long',
+          day: 'numeric',
+        }),
+      },
+    });
+    render(<CourseMain courseName="Node.js course" type="Mentoring Program" />);
+    const labelElement = screen.getByText('available');
+    expect(labelElement).toBeVisible();
+  });
+});
```

Comment:
Maybe we should add more test cases to test also 'upcoming' courses

Reply:
Done